arrayWithRandomValues.java

```diff
@@ -0,0 +1,69 @@
+public class arrayWithRandomValues {
+    public static void main(String[] args){
+        int []array = array();
+        average(array);
+        min(array);
+        max(array);
+        bubble(array);
+
+    }
+
+
+    public static int[] array(){
+        int array[] = new int[10];
+        for (int i = 0; i < array.length; i++) {
+            array[i] = (int) (Math.random() * 20);
+            System.out.println(array[i]);
+
+        }
+        return array;
+    }
+
+        public static double average(int []array){
+
+            double average = array[0];
+            for (int i = 0; i < array.length; i++) {
+            average += array[i];
+        }
+        average /= 10;

```

Comment:
What is magic number 10? What if your method will receive array with more than 10 values. You should use here array.length